### DISCUSSION TITLE
LibreHardwareMonitor v0.9.2 & IPMI

### DISCUSSION THREAD

**Discussion by Minxster** [Rem0o/FanControl.Releases#1683] [General]
I'm wondering, with the latest update to LibreHardwareMonitor, will you be implementing the option for IPMI? Hopefully both reading fan and temperature (and more) information, along with controlling the fans?

I appreciate that there will be a small use-case for this functionality, since just about all computers don't have IPMI. But it would really help the home-server community?

If this is an option, or maybe in the future - I'd love to be able to read other values (as temperatures) into FanControl - though these would be grabbed from a SSH script to read drive temperatures from my servers; e.g. reading 24 disk temperatures per server. Though the act of SSH'ing into a server and running "a script" will fall out of scope for FanControl (I presume), but having some JSON/CSV reading function (I have no idea if one already exists) would be really useful. This, along with IPMI, would really make FanControl a go-to for server control IMO 🤷‍♂️😁

All of this ties back to a script being used by (some) TrueNAS folk, which can work perfectly for server-based systems 🤷‍♂️
https://www.truenas.com/community/resources/fan-scripts-for-supermicro-boards-using-pid-logic.24/

Anywho, this is just some thoughts and hopes 😉😊

**Comment by Rem0o** [comment]
FanControl pulls from LHM directly, so it will/already have the IPMI stuff in the dll. As for injection temperatures from external sources, you can either use a "File" custom sensor, or a plugin if some more complex logic is required.
